Sprint-1/exercises/count.js

```diff
@@ -2,5 +2,8 @@ let count = 0;
 
 count = count + 1;
 
+console.log(count)
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
 // Describe what line 3 is doing, in particular focus on what = is doing
+//In line 3 , count is adding argument 1 to initial count with value of 0 then the result that is 1 assign
+// to the count. new value! 
```

Comment:
- Suppose value of count was obtained from a user (so it can be any number). How would you describe line 3 for such general case?
- May I suggest feeding the code to ChatGPT for a second opinion or to find out how else we can describe/explain a statement. From time to time you may learn some terminology or more concise way to describe code.

Reply:
We can also just say "increase/increment the value of `count` by 1"